### DISCUSSION TITLE
Did the 5.0.2 release get modified after being posted?

### DISCUSSION THREAD

**Discussion by einheber-seed** [cmusphinx/pocketsphinx#366] [Q&A] [answered]
Asking because the sha seems to have changed.

**Comment by dhdaines** [comment, marked as answer]
Yes, sorry about that, the initial GitHub release was a misfire.  The current one is correct, as is everything on PyPI.

I don't take GitHub releases, which are just glorified tags as far as I can tell, very seriously (perhaps I should?).  When it's on PyPI, it's for real.
